pkg/machinery/client/events.go

```diff
@@ -47,6 +47,13 @@ func WithTailDuration(dur time.Duration) EventsOptionFunc {
 	}
 }
 
+// WithTailActorID sets up Watcher to return events with the specified actor ID.
```

Comment:
```suggestion
// WithActorID sets up Watcher to return events with the specified actor ID.
```

Reply:
Not sure about this because it's in the client package and `client.WithActorID` doesn't tell much. All other tail builder functions follow the `WithTail*` naming. `WithTailEvents`, `WithTailID`, `WithTailDuration`.